deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkProcessor.java

```diff
@@ -116,13 +113,14 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
         = !roundEnv.getElementsAnnotatedWith(DeepLinkHandler.class).isEmpty();
 
     if (!deepLinkElements.isEmpty()) {
+      String packageName = deepLinkElements.get(0).getPackageName();
```

Comment:
I'm not super familiar with this code - are all of the items called in `process` from the same package? Are there possibly more than one package per `process` call?